docs/VLCB4in4out_README.md

```diff
@@ -80,18 +79,21 @@ OP_CODE | HEX | Function
  OPC_ASON | 0x98 | Short event on
  OPC_ASOF | 0x99 | Short event off
 
-### Event Variables
+## Set Up
+
+The module is initialised by pressing the VLCB button (formerly CBUS button) for
+6 seconds when the green LED goes off and the Yellow LED flashes. A module name and
+Node Number can then be set via the FCU in the normal manner.
 
-Event Variable 1 (EV1) is reserved for system use to identify the source of
-produced event.
+## Event Variables
```

Comment:
I don't think you need a heading for "Event Variables". The sub-section on produced events don't mention event variables. Maybe call this heading "Events" only?

Reply:
Good point.